src/frontend/src/molecules/PostCard.jsx

```diff
@@ -26,10 +26,21 @@ const PostCard = ({ post }) => {
     }
   }
 
+  function toggleFavorite(e) {
+    e.preventDefault()
+    const heart = e.target
+    console.log(heart.className)
+    if (heart.className === "far fa-heart") {
+      heart.className = "fas fa-heart"
+    } else if (heart.className === "fas fa-heart") {
+      heart.className = "far fa-heart"
+    }
```

Comment:
It'd be great if we include the `else` statement as well to handle the corner case (if any).

Reply:
Good idea.